components/Snippet.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Highlight from 'react-highlight';
+require('style!css!highlight.js/styles/github-gist.css');
```

Comment:
Here, https://github.com/webpack/webpack.js.org/blob/develop/webpack.config.js#L9 . You can point from the entry.